pymongo/event_listeners.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from pymongo import monitoring
+
+
+class CommandListener(monitoring.CommandListener):
+
```

Comment:
These commands need docstrings explaining what they do. See for example: https://github.com/mongodb/mongo-python-driver/blob/58aaede0fe77e1f579b1a203cebffea91b9f66e8/pymongo/monitoring.py#L415 . The `.. versionaddded` would be 3.11.

A file corresponding to this new submodule also needs to be added to the API docs here: https://github.com/mongodb/mongo-python-driver/tree/master/doc/api/pymongo